tsconfig.json

```diff
@@ -6,7 +6,15 @@
 		"target": "es6",
 		"lib": [
 			"es2015"
-		]
+		],
+		//"strict": true,
+		"noImplicitAny": true,
+		"noImplicitThis": true,
+		"alwaysStrict": true,
+		"strictBindCallApply": true,
+		"strictFunctionTypes": true,
+		//"strictNullChecks": true,
+		//"strictPropertyInitialization": true,
```

Comment:
can we add an issue to enable the remaining checks for --strict ?

Reply:
Indeed, I can do so once we finish this PR